packages/reports/src/types.ts

```diff
@@ -0,0 +1,27 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2020 Beyond Essential Systems Pty Ltd
+ */
+
+import { Request } from 'express';
+import { ParamsDictionary, Query } from 'express-serve-static-core';
+import { AccessPolicy } from '@tupaia/access-policy';
+import { Authenticator } from '@tupaia/auth';
+import { TupaiaDatabase, ModelRegistry } from '@tupaia/database';
+
+export interface ReportsRequest<
+  P = ParamsDictionary,
+  ResBody = any,
+  ReqBody = ReportsRequestBody,
+  ReqQuery = Query
```

Comment:
Obviously we will have to move the definitions for `FetchReportParams`, `ReportsRequestBody` from `fetchReports.ts` to here. It may also be worth re-ordering the definitions according to usage (eg `FetchReportParams`, `ReportsRequestBody` etc goes first, then the dependent `ReportsRequest` goes last)

Reply:
Is it okay if I address this in my next PR? I've already done half of that refactor there already

Reply:
No problem at all! 🙂